src/imitation/scripts/config/train_adversarial.py

```diff
@@ -259,3 +264,9 @@ def fast():
         max_episode_steps=500,  # To match `inverse_rl` settings.
     ),
 )
+
+mujoco_shared_locals = dict(
```

Comment:
These are constant so make them uppercase. Ditto for `ant_shared_locals`. I'd also prefer if they were at the top of the file so can read file top-to-bottom and understand it in one pass, but don't have strong feeling here.